scripts/lxc-to-lxd

```diff
@@ -128,6 +212,16 @@ def convert_container(lxd_socket, container_name, args):
     # As some keys can't be queried over the API, parse the config ourselves
     print("Parsing LXC configuration")
     lxc_config = config_parse(container.config_file_name)
+    found_keys = config_keys(lxc_config)
+
+    print("Checking for unsupported LXC configuration keys")
+    diff = list(set(found_keys) - set(supported_keys))
+    for d in diff:
+        if (not d.startswith('lxc.network.') and not
+                d.startswith('lxc.cgroup.')):
+            print("Found at least one unsupported config key: ", d)
```

Comment:
Also, the skip messages are usually one line, so I think we'd want:

    print("Found at least one unsupported config key, "%s", skipping..." % d)